src/plugins/viewLargeAction/viewLargeAction.js

```diff
@@ -99,7 +99,10 @@ export default class ViewLargeAction {
       }
     );
     this.preview = vNode.componentInstance;
-    this.destroy = destroy;
+    this.destroy = () => {
```

Comment:
Not really applicable to this PR, but it looks like PreviewContainer throws a Vue warning when reusing an existing component (e.g., using viewLargeAction in a DisplayLayout):
```
[Vue warn]: Maximum recursive updates exceeded. This means you have a reactive effect that is mutating its own dependencies and thus recursively triggering itself. Possible sources include component template, render function, updated hook or watcher source function.

... repeating a zillion times ...
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
flushJobs @ runtime-core.esm-bundler.js:372
Promise.then (async)
queueFlush @ runtime-core.esm-bundler.js:275
queuePostFlushCb @ runtime-core.esm-bundler.js:295
queueEffectWithSuspense @ runtime-core.esm-bundler.js:1665
componentUpdateFn @ runtime-core.esm-bundler.js:5770
run @ reactivity.esm-bundler.js:178
instance.update @ runtime-core.esm-bundler.js:5861
setupRenderEffect @ runtime-core.esm-bundler.js:5869
mountComponent @ runtime-core.esm-bundler.js:5659
processComponent @ runtime-core.esm-bundler.js:5612
patch @ runtime-core.esm-bundler.js:5087
mountChildren @ runtime-core.esm-bundler.js:5331
mountElement @ runtime-core.esm-bundler.js:5238
processElement @ runtime-core.esm-bundler.js:5203
patch @ runtime-core.esm-bundler.js:5075
mountChildren @ runtime-core.esm-bundler.js:5331
mountElement @ runtime-core.esm-bundler.js:5238
processElement @ runtime-core.esm-bundler.js:5203
patch @ runtime-core.esm-bundler.js:5075
componentUpdateFn @ runtime-core.esm-bundler.js:5755
run @ reactivity.esm-bundler.js:178
instance.update @ runtime-core.esm-bundler.js:5861
setupRenderEffect @ runtime-core.esm-bundler.js:5869
mountComponent @ runtime-core.esm-bundler.js:5659
processComponent @ runtime-core.esm-bundler.js:5612
patch @ runtime-core.esm-bundler.js:5087
mountChildren @ runtime-core.esm-bundler.js:5331
mountElement @ runtime-core.esm-bundler.js:5238
processElement @ runtime-core.esm-bundler.js:5203
patch @ runtime-core.esm-bundler.js:5075
componentUpdateFn @ runtime-core.esm-bundler.js:5755
run @ reactivity.esm-bundler.js:178
instance.update @ runtime-core.esm-bundler.js:5861
setupRenderEffect @ runtime-core.esm-bundler.js:5869
mountComponent @ runtime-core.esm-bundler.js:5659
processComponent @ runtime-core.esm-bundler.js:5612
patch @ runtime-core.esm-bundler.js:5087
componentUpdateFn @ runtime-core.esm-bundler.js:5755
run @ reactivity.esm-bundler.js:178
instance.update @ runtime-core.esm-bundler.js:5861
setupRenderEffect @ runtime-core.esm-bundler.js:5869
mountComponent @ runtime-core.esm-bundler.js:5659
processComponent @ runtime-core.esm-bundler.js:5612
patch @ runtime-core.esm-bundler.js:5087
render @ runtime-core.esm-bundler.js:6379
mount @ runtime-core.esm-bundler.js:3832
app.mount @ runtime-dom.esm-bundler.js:1449
mount @ mount.js:10
_getPreview @ viewLargeAction.js:81
_expand @ viewLargeAction.js:65
invoke @ viewLargeAction.js:51
action.onItemClicked @ MenuAPI.js:81
callWithErrorHandling @ runtime-core.esm-bundler.js:158
callWithAsyncErrorHandling @ runtime-core.esm-bundler.js:166
invoker @ runtime-dom.esm-bundler.js:595
eventHelpers.js:38 [Violation] Added non-passive event listener to a scroll-blocking 'wheel' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
listenTo @ eventHelpers.js:38
initCanvas @ MctPlot.vue:885
callWithErrorHandling @ runtime-core.esm-bundler.js:158
callWithAsyncErrorHandling @ runtime-core.esm-bundler.js:166
emit @ runtime-core.esm-bundler.js:669
visibilityChanged @ MctChart.vue:300
```

Seems to happen with any type of component in a DisplayLayout, so probably an issue with PreviewContainer.

Reply:
https://github.com/nasa/openmct/issues/7488